action.yml

```diff
@@ -1,108 +1,41 @@
-  using: composite
-  steps:
-    - name: Check If Template
-      id: template
-      run: |
-        template=$(awk -F'=' '/^IS_LIBRARY:=/{print $2}' Makefile)
-        echo $template
-        echo "template=$template" >> $GITHUB_OUTPUT
-      env:
-        TEMPLATE: ${{ steps.template.outputs.template }}
-      shell: bash
-
-    - name: Get Project Info
-      if: ${{ steps.template.outputs.template == 1 && inputs.library-path != null }}
-      id: project-info
-      shell: bash
-      run: |
-        if [ $ACTION = opened ]; then
-            # we have to get the head sha directly from the pr api, because github silly ig
-            echo "then"
-            sha=$(wget -O- --quiet https://api.github.com/repos/"${{inputs.repository}}"/pulls/$PR_NUM | jq -r .head.sha | head -c 6)
-        else
-            echo "else"
-            sha=$(echo "$AFTER" | head -c 6)
-        fi
-
-        echo "sha=$sha" >> $GITHUB_OUTPUT
-        echo "SHA found: $sha"
-
-        version=$(awk -F'=' '/^VERSION:=/{print $2}' Makefile)
-        echo "Version found: $version"
-        echo "version=$version" >> "$GITHUB_OUTPUT"
-
-        postfix="$version+$sha"
-        echo "postfix=$postfix" >> "$GITHUB_OUTPUT"
-        echo "Postfix found: $postfix"
-
-        library_name=$(awk -F'=' '/^LIBNAME:=/{print $2}' Makefile)
-        echo "library_name=$library_name" >> "$GITHUB_OUTPUT"
-        echo "Library name found: $library_name"
-
-        name="$library_name@$postfix"
-        echo "name=$name" >> "$GITHUB_OUTPUT"
-        echo "Name found: $name"
-
-      env:
-        AFTER: ${{ github.event.after }} # this isn't present for pr opened events
-        ACTION: ${{github.event.action}}
-        PR_NUM: ${{github.event.number}}
-
-    - name: Install ARM Toolchain
-      uses: carlosperate/arm-none-eabi-gcc-action@v1
-      with:
-        release: "10-2020-q4"
-
-    - name: Build Project
-      run: |
-        make clean quick -j
-      shell: bash
-
-    - name: Prepare Template
-      if: ${{ steps.template.outputs.template == 1 && inputs.library-path != null }}
-      shell: bash
-      run: |
-        # Update version in Makefile
-        sed -i "s/^VERSION:=.*\$/VERSION:=${{steps.project-info.outputs.postfix}}/" Makefile
-        cat Makefile
-
-        # fake pros c create-template for make template
-        PATH="$PATH:$GITHUB_ACTION_PATH/pros-fake/bin"
-        
-        make template
-
-        mkdir -p template/include/"${{inputs.library-path}}"/
-
-        cp {LICENSE*,README*} template/include/"${{inputs.library-path}}"/
-
-        echo "\n## [Github link](${{github.server_url}}/${{github.repository}})" >> template/include/"${{inputs.library-path}}"/README.md
-        perl -i -pe 's@(?<=[^/])(docs/assets/.*?)(?=[")])@${{github.server_url}}/${{github.repository}}/blob/master/$1?raw=true@g' template/include/"${{inputs.library-path}}"/README.md
-        echo ${{steps.project-info.outputs.postfix}} >> template/include/${{inputs.library-path}}/VERSION
```

Comment:
It also doesn't do this, since I figured that these three lines were LemLib specific (and therefore wouldn't really be beneficial to do in a more general project)